CHANGES.rst

```diff
@@ -257,6 +257,8 @@ Bug Fixes
 
 - ``astropy.table``
 
+  - Fix reading of big ascii tables with the fast reader. [#5319]
```

Comment:
Being more specific would be useful for users.  I assume the cutoff is 4 Gb file length?  ASCII should be upper case here.  Also, this should be in the `astropy.io.ascii` section, not table.